scripts/install-mender.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/sh
+
+set -e
+
+CHANNEL="stable"
+
+DEFAULT_COMPONENTS="mender-client"
+COMPONENTS="$DEFAULT_COMPONENTS mender-shell"
+SELECTED_COMPONENTS=$DEFAULT_COMPONENTS
+
+export DEBIAN_FRONTEND=noninteractive
+
+banner (){
+    echo "
+                          _
+ _ __ ___   ___ _ __   __| | ___ _ __ 
+| '_ \` _ \ / _ \ '_ \ / _\` |/ _ \ '__|
+| | | | | |  __/ | | | (_| |  __/ |   
+|_| |_| |_|\___|_| |_|\__,_|\___|_|   
+
+Running the Mender installation script.
+--
+"
+                                      
+}
+
+usage() {
+    echo "usage: install-mender.sh [-h] [-c channel] [component...]"
+    echo "  -h             print this help"
+    echo "  -c             channel: stable(default)|experimental"
+    echo "  <component>    list of components to install"
+    echo ""
+    echo "Supported components:"
+    echo "  * mender-client (default if none provided)"
+    echo "  * mender-shell"
```

Comment:
[MEN-4150](https://tracker.mender.io/browse/MEN-4150) says both should be installed. Should it be changed?

Reply:
By default we should install all (see my yesterday's comment). Then we will be fulfilling the acceptance criteria, while having options to actually test it today (we cannot install both as per today, as I tried to explain also in my comment).

Reply:
oh yes, sorry, that was a git push mistake.
now it's `+DEFAULT_COMPONENTS="mender-client mender-shell" `.